src/gwbot/include/gwbot/lib/tg_event.h

```diff
@@ -412,19 +408,19 @@ static __always_inline int parse_tgevi_chat(json_object *jchat,
 		}
 	} else {
 		if (chat->type == TGEV_CHAT_GROUP) {
-			if (unlikely(!json_object_object_get_ex(jchat, 
-					"all_members_are_administrators", 
+			if (unlikely(!json_object_object_get_ex(jchat,
+					"all_members_are_administrators",
 								&res))) {
 				pr_err("Cannot find \"all_admins\" "
 						"on key \"chat\"");
 				return -EINVAL;
 
 			}
-			chat->all_admins = 
+			chat->all_admins =
 				json_object_get_boolean(res)  ? true : false;
 		}
```

Comment:
Doesn't `json_object_get_boolean` return `bool`?
Do we need this ternary check?